public/tutorials/setup-for-testing.sh

```
@@ -0,0 +1,415 @@
+#! /bin/sh
+
+rm -rf testing
+mkdir -p testing/forge
+cd testing/forge
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)